src/declarations/stencil-public-compiler.ts

```diff
@@ -42,6 +42,40 @@ export interface StencilConfig {
    * To disable this feature, set enableCache to false.
    */
   enableCache?: boolean;
+  /**
+   * The directory where sub-directories will be created for caching when `enableCache` is set
+   * `true` or if using Stencil's Screenshot Connector.
+   *
+   * @default '.stencil'
+   *
+   * @example
+   *
+   * A Stencil config like the following:
+   * ```ts
+   * export const config = {
+   *  ...,
+   *  enableCache: true,
+   *  cacheDir: '.cache',
+   *  testing: {
+   *    screenshotConnector: 'connector.js'
+   *  }
+   * }
+   * ```
+   *
+   * Will result in the following file structure:
+   * ```tree
+   * stencil-project-root
+   * └── .cache
+   *     ├── .build <-- Where build related file caching is written
+   *     |
+   *     └── screenshot-cache.json <-- Where screenshot caching is written
+   * ```
+   */
+  cacheDir?: string;
+  /**
+   * @deprecated this flag is no longer used by Stencil and can be safely removed.
+   */
+  enableCacheStats?: boolean;
```

Comment:
Do we want to just remove this now since we're just adding this change to v4? It's not used anywhere so the only "breaking change" is that TS will throw an error if it was in a project's config

Reply:
I'm down 👍

Reply:
I'm OK with it, so long as we can get everything documented/landed by Monday